### DISCUSSION TITLE
Table Not Showing

### DISCUSSION THREAD

**Discussion by kb6673** [rappasoft/laravel-livewire-tables#767] [General]
I've used Livewire tables for a while now (v1 and some v1 -> v2 upgrades), however this is my first new project using v2. I've got a fairly clean Lara 9 install and Livewire Tables v2 install.

Setup all went fine. When I add `<livewire:tables.courses-table />` I can see the queries and views in the DebugBar

![image](https://user-images.githubusercontent.com/1001484/167476395-b486e781-21c8-4a7c-9083-8ce527a9dd54.png)

However no table is shown, not even the empty "no results found".

This is what I see in the HTML

![image](https://user-images.githubusercontent.com/1001484/167476506-6b4c0f87-a7ce-4b5f-aa46-59f4e2f03fd6.png)

I'm very clearly doing something wrong, however I can't see what! Can anyone slap me with the obvious?

**Comment by inmanturbo** [comment]
Can you share the code for your table component? and the markup where you are rendering it?
